configure.in

```
@@ -515,7 +515,7 @@ fi
 # but has its own.  Also check other compiler-specific flags here.
 
 if test "$GCC" = yes -a "$ICC" = no; then
-  CFLAGS="-Wall -Wmissing-prototypes -Wpointer-arith"
+  CFLAGS="-Wall -Wmissing-prototypes -Wpointer-arith -Wimplicit-fallthrough=3 -Wvla"
```

Maybe still ignore `-Wno-deprecated-declarations`?
```suggestion
  CFLAGS="-Wall -Wmissing-prototypes -Wpointer-arith -Wimplicit-fallthrough=3 -Wvla -Wno-deprecated-declarations"
```